packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -499,6 +499,8 @@ const registerNetworkIpc = (mainWindow) => {
       let response, responseTime;
       try {
         /** @type {import('axios').AxiosResponse} */
+        // Looks like the "params" create problems if filled
+        request.params = [];
```

Comment:
After interpolation, we can remove the params values as they are already interpolated in the URL, making the approach simpler.

https://github.com/usebruno/bruno/blob/9b382fa0911536a0bfa27ae0d8a67007cc1364f7/packages/bruno-electron/src/ipc/network/interpolate-vars.js#L118

`  delete request.params;
`

Reply:
@sanjai0py Thanks a lot for the hint (I'm still working on my knowledge of Bruno), I've just updated my PR. Let me know if it's ok, hope to be (somehow) helpful